packages/liveblocks-client/package.json

```diff
@@ -14,6 +14,7 @@
     "build": "tsup",
     "format": "(eslint --fix src/ || true) && prettier --write src/",
     "lint": "eslint src/",
+    "lint:package": "npx -y publint@latest --strict && npx -y @arethetypeswrong/cli@latest /tmp/$( npm pack -q --pack-destination /tmp )",
```

Comment:
We may consider installing these tools as top-level dev dependencies:

https://github.com/liveblocks/liveblocks/blob/main/package.json#L18-L25

But I've deliberately set this up to always use the latest version, because these tools are still in development, and if new best-practices arise, we'd always see the latest recommendations here. If we think it's not a good idea, I'm happy to pin this dependency explicitly too.

Reply:
This morning, the latest version of `attw` broke when running on Node 16. To avoid breaking our builds, I've pinned the versions explicitly anyway, see d53d4ca. Let's just upgrade them regularly to stay up to date instead.